VELOCITY_RESOLUTION: 0.03
YAWRATE_RESOLUTION: 0.03
PREDICT_TIME: 8.0
ANGLE_RESOLUTION: 0.0087

MAX_VELOCITY: 0.5
MIN_VELOCITY: -0.5
MAX_YAWRATE: 1.50
MAX_D_YAWRATE: 2.4
MAX_ACCELERATION: 1.8
HZ: 10
